tmt/utils.py

```diff
@@ -1079,6 +1113,26 @@ def default_branch(repository, remote='origin'):
         return ref.read().strip().split('/')[-1]
 
 
+def parse_dotenv(content: str) -> Dict[str, str]:
+    """ Parse dotenv (shell) format of variables """
+    try:
+        return dict([line.split("=")
+                    for line in shlex.split(content, comments=True)])
+    except ValueError:
+        raise GeneralError(f"Failed to extract variables from:\n{content}"
```

Comment:
If we make this exception less verbose, then how the attacker can see the content of a file?

Reply:
Good point. IMHO Exception should just mentioned the path.

Reply:
Should be addressed by af814e6.